index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zuwairanajma Web</title>
+    <script src = "https://kit.fontawesome.com/1935d064dd.js" 
+    crossorigin = "anonymous">
+   </script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!-- Toolbar Header Section starts-->
+    <header> 
+        <div id= "head-wrap" class="headbar">
+    
+  <nav class = "headbar-menu">
+    <p class=""logo>My Logo</p>
```

Comment:
- [ ] I suggest using an anchor tag (`a`) for your logo, that'll navigate to the home page. That way your project will become more semantic & improve UI.